### ISSUE TITLE
[Feature Request] Analog to face buttons

### ISSUE THREAD

**Issue by arnoldsecret** [libretro/RetroArch#8410]
SNES ports of games like Libble Rabble and Smash T.V. as well as certain PSP games like The 3rd Birthday and Coded Arms can benefit from this.

**Comment by orbea** [comment]
Can you elaborate what you are talking about? I'm not very familiar with the games or the use case. There is also an issue template for a reason...

**Comment by fr500** [comment]
He means playing it as a twin stick shooter.
You can achieve this using remapping so no need for a niche feature for
this.

**Comment by orbea** [comment]
I understand now and yes remapping seems like it should suffice.

@arnoldsecret Does this work for you?

**Comment by grant2258** [comment]
Well only issue I could see with this is the analog would need at least a little pressure on it before acknowledging a button press. Again im not sure how analog remaps to digital the slightest touch shouldnt activated it

**Comment by fr500** [comment]
it uses the same threshold as d-pad to analog.

**Comment by grant2258** [comment]
well id say its down to the core to map it properly or remap via controls have to agree with you @fr500 . if the threshold is already in place there is no need for anything to be done. Do you have to have the activate analog to digital for this remapping to work?

**Comment by fr500** [comment]
No, you have to do manual remapping of the buttons to analog axes.
